app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectInformationActivity.kt

```diff
@@ -319,6 +319,15 @@ class ProjectInformationActivity : AppCompatActivity() {
         }
     }
 
+    private fun getDynamicLink() : Uri {
```

Comment:
I would just suggest changing the name of this function to createDynamicLink() for clarity but this is unimportant.

Reply:
I agree the name suggested is more clear!